### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/1-index](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes following components:

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Custom-Directories](https://hosted.weblate.org/projects/jamulus/custom-directories/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/footertext](https://hosted.weblate.org/projects/jamulus/footertext/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/general](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: _translator-files/po/fr/Tips-Tricks-More.po [hunk 1]
```diff
@@ -110,8 +110,8 @@ msgstr ""
 #, no-wrap
 msgid "**NOTE:** This does not stop people from hearing you, using the chat function, or seeing your profile information.\n"
 msgstr ""
-"**NOTE :** Cela ne les empêche pas de vous entendre, d'utiliser la fonction "
-"chat ou de voir les informations de votre profil.\n"
+"**NOTE :** cela ne les empêche pas de vous entendre, d'utiliser la fonction "
+"tchate ou de voir les informations de votre profil.\n"
```
Comment (jujudusud) [inline _translator-files/po/fr/Tips-Tricks-More.po:114]: Thanks, better like that!

File: _translator-files/po/fr/Tips-Tricks-More.po [hunk 2]
```diff
@@ -305,7 +305,7 @@ msgstr "## Pour les administrateurs de serveurs"
 #: ../wiki/en/Tips-Tricks-More.md:104
 msgid "### Converting a Registered Server to an Unregistered one on the fly"
 msgstr ""
-"### Convertir d'un serveur enregistré en serveur non enregistré à la volée"
+"### Convertir un serveur enregistré en serveur non enregistré à la volée"
```
Comment (jujudusud) [inline _translator-files/po/fr/Tips-Tricks-More.po:308]: Thanks. Nice like that.

**Review by jujudusud — APPROVED** [review]

File: _translator-files/po/fr/1-index.po [hunk 3]
```diff
@@ -6,14 +6,16 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"PO-Revision-Date: 2022-08-04 01:33+0200\n"
-"Last-Translator: \n"
-"Language-Team: \n"
+"PO-Revision-Date: 2022-08-15 15:17+0000\n"
+"Last-Translator: Julien Taverna <jujudusud@gmail.com>\n"
```
Comment (ann0see) [inline _translator-files/po/fr/1-index.po]: @jujudusud @ignotus666 Is it ok to have E-Mails here?
I'd rather not like to have mine in here

**Review by ann0see — COMMENTED** [review]
Interesting.

File: _translator-files/po/de/Installation-for-Linux.po [hunk 4]
```diff
@@ -6,14 +6,16 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"PO-Revision-Date: 2022-06-20 12:01+0200\n"
-"Last-Translator: \n"
-"Language-Team: \n"
+"PO-Revision-Date: 2022-08-15 20:17+0000\n"
+"Last-Translator: ann0see <ann0see@t-online.de>\n"
```
Comment (ann0see) [inline _translator-files/po/de/Installation-for-Linux.po:10]: Yeah. As expected...

File: _translator-files/po/fr/1-index.po (hunk 3 above)
Comment (jujudusud) [reply _translator-files/po/fr/1-index.po]: I don't know if it is a good or a bad things.

Comment (ann0see) [reply _translator-files/po/fr/1-index.po]: I see it as a bit sub optimal (spam etc). My E-Mail can be found in my GH profile in encoded form, but we don't know if all contributors want their Mail being displayed here.

**Comment by ann0see** [comment]
I'm not sure, but probably we could even merge this to release?

**Comment by ignotus666** [comment]
I think so. But maybe merge it into next-release first and then from there into release to avoid weird things happening with weblate's branch. It might not matter, but just in case.

**Comment by ann0see** [comment]
No. The cleanest way is release->next release --> force update Weblate, I think.

**Comment by ignotus666** [comment]
Ok, that should work.

**Comment by ann0see** [comment]
Hmm. No. There are some other unrelated changes on next-release. So I'll merge it to next-release.

**Comment by ann0see** [comment]
I'll take care of the E-Mail tomorrow manually.

**Comment by ignotus666** [comment]
I wonder why it still says there are 52 outgoing commits on weblate - it should have been reset to zero, shouldn't it?

**Comment by ann0see** [comment]
It needs to be force reset.

**Comment by ann0see** [comment]
Done

**Comment by ignotus666** [comment]
Will it always have to be forced? I didn't have to do that in my tests.

**Comment by ann0see** [comment]
Only if we squash the commits. It was certainly needed here, since there were way too many small commits.

**Comment by ann0see** [comment]
I think fixing conflicts on the weblate repo is more difficult. I now did it on GitHub the standard way.

**Pull request merged**